bokeh/io/export.py

```diff
@@ -30,7 +30,7 @@
 
 # Bokeh imports
 from ..embed import file_html
-from ..resources import INLINE
+from ..resources import LEGACY_INLINE
```

Comment:
Suggest `INLINE_LEGACY` or `INLINE_ES5`

Reply:
I can rename it, but in any case, this is just a temporary measure to make exports' tests pass. I will be redesigning resources, and having separate definition for legacy inline resources most likely won't be necessary.